platform/ios/MGLMapboxEvents.m

```diff
@@ -430,9 +435,25 @@ - (void) pushEvent:(NSString *)event withAttributes:(NSDictionary *)attributeDic
     __weak MGLMapboxEvents *weakSelf = self;
 
     dispatch_async(_serialQueue, ^{
+
         MGLMapboxEvents *strongSelf = weakSelf;
+
         if ( ! strongSelf) return;
-        
+
+        if ([MGLEventTypeAppUserTurnstile isEqualToString:event]) {
```

Comment:
Thanks for taking my earlier feedback into account. Now that I look at this, I’m thinking it would be cleaner to have a separate, specialized method called `-pushTurnstileEvent` just for this chunk of code (and all the weak/strong/dispatch scaffolding above). That way there are no special cases and no possibility of somehow accidentally turning the turnstile more than once.

Not critical though: I think this PR looks good overall.